src/StorSimple/Microsoft.WindowsAzure.Management.StorSimple.nuspec

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata minClientVersion="2.5">
+    <id>Microsoft.WindowsAzure.Management.StorSimple</id>
+    <title>Microsoft Azure StorSimple Management Library</title>
+    <releaseNotes>__BASELINE_RELEASE_NOTES__</releaseNotes>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <owners>azure-sdk, Microsoft</owners>
+    <licenseUrl>http://aka.ms/windowsazureapache2</licenseUrl>
+    <projectUrl>https://github.com/Azure/azure-sdk-for-net</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288890</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>Provides capabilities to manage Microsoft Azure StorSimple.</summary>
+    <description>Provides capabilities to manage Microsoft Azure StorSimple.</description>
+    <copyright>Copyright © Microsoft Corporation</copyright>
+    <tags>Microsoft Azure Management StorSimple REST HTTP client windowsazureofficial</tags>
+    <references>
+      <group targetFramework="portable-net45+sl50+wp80+win">
+        <reference file="Microsoft.WindowsAzure.Management.StorSimple.dll" />
+      </group>
+      <group targetFramework="net40">
+        <reference file="Microsoft.WindowsAzure.Management.StorSimple.dll" />
+      </group>
+    </references>
+    <dependencies>
+      <dependency id="Microsoft.WindowsAzure.Common" version="[1.4.0,2.0)" />
```

Comment:
All new packages should take a dependency on Microsoft.Azure.Common `[2.0.1, 3.0)`